syne_tune/optimizer/schedulers/synchronous/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+__all__ = ['SynchronousHyperbandScheduler',
```

Comment:
I am not sure we want to maintain a list of imports in the root of every sub folder for schedulers (async, sync, multiobj, etc). We have a list of method definitions in baselines, should we keep this one as the main entry point?

Reply:
I'd prefer to have this as well. It is optional in a way for whoever adds something there.
But quite some users will still want to create things not using the baselines. And in fact, the baseline definitions should also be easy to understand, so they should use the simplified imports as well.

Reply:
If the backends get easier imports, the schedulers should get them as well, both are key concepts that can become many.
Some people will just not like to use baselines, and some scheduler combinations may not even be in there.